announcements/_posts/2019-04-05-w23-customization.md

```diff
@@ -0,0 +1,38 @@
+--
+author: Jeffery Russell (@Jrtechs)
+title: "Week #11: Workflow Customization"
+layout: post-event
+date-start: "2019-04-12 16:00"                               
+date-end: "2019-04-12 18:00"
+location: "GOL-2650 (Lg. DB Lab)"
+---
+
+Hey Ritluggers,
+
+Do you want to learn how to pimp your workflow?
+This week we will be doing a ton of mini demos on how to customize everything from zsh to git.
+This is a great opportunity to learn some amazing tricks and hacks to make your workflow more stylish and convenient.
+Be there or be stock bash.
+
+This Friday(4-12) is the last day for eboard nominations.
+If you feel like helping run RITlug next semester while building your resume, [sign up using this link.](http://cglink.me/s17020).
+Elections will open on Monday.
+
+Our semesterly FOSS Family Dinner is around the corner!
+This is a great chance to meet alumni and other members of the FOSS community around RIT and Rochester.
+This semester's dinner will be taking place on the last Wednesday of the month, April 24th, starting at 6:00pm at Sticky Lips!
+Please follow [this link](https://www.eventbrite.co.uk/e/foss-family-dinner-sticky-lips-tickets-59863140264) to RSVP.
+
+Hope to see you all at this week's meeting.
+
+Jeffery Russell @Jrtechs
+
+---
+
+#### News
+
+* [Red Hat will be working to bring Fedora to arm64 laptops](https://fossbytes.com/red-hat-fedora-bring-linux-arm-laptops/)
+* [EU's new copyright rule, Article 13](https://www.cnn.com/2019/03/26/tech/eu-copyright-article-13/index.html)
```

Comment:
this was on last week's email as well

Reply:
The Red Hat one is new. I want to keep the article 13 one on there because it is a huge story